doc/mgr/rgw.rst

```diff
@@ -166,7 +164,7 @@ Upgrading root ca certificates
 
       openssl x509 -in <cert_name>.crt -noout -text
 
-#. Perform the above steps on the MGR node and on the RGW node of all secondary
+#. Perform the above steps on the Manager node and on the RGW node of all secondary
```

Comment:
active Manager node
RGW nodes

I suspect

Reply:
Kind of a rabbithole.  If we speak of DNF then we confuse people running releases that don't call it DNF yet.